bip-musig2.mediawiki

```diff
@@ -200,6 +207,7 @@ The following conventions are used, with constants as defined for [https://www.s
 *** Return the unique point ''P'' such that ''x(P) = x'' and ''y(P) = y''.
 ** The function ''point(x)'', where ''x'' is a 32-byte array ("X-only" serialization), returns ''lift_x(int(x))''. Fail if ''lift_x'' fails.
 ** The function ''cpoint(x)'', where ''x'' is a 33-byte array (compressed serialization), sets ''P = lift_x(int(x[1:33]))'' and fails if that fails. If ''x[0] = 2'' it returns ''P'' and if ''x[0] = 3'' it returns ''-P''. Otherwise, it fails.
```

Comment:
How would you feel about using a less likely byte array as the serialization of infinity, since using all zeros could cause a fresh byte array to accidentally get interpreted as infinity?

Maybe something like `FFFF...F` instead?

Reply:
Hm all zeros seems the most straightforward (and closer to the serialization in the [SEC standard](https://www.secg.org/SEC1-Ver-1.0.pdf) which is just a single `0x00` byte. A fresh byte array is unitialized in C and in rust the only way to create an array is by specifying its contents. Is `FFFF...F` mod field order a valid X coordinate? If so, we could accidentally introduce other problems.

Reply:
Given your reasoning plus the fact that 0 is not a valid X coordinate (but 2^256 - 1 is, mod n), keeping it as all 0's makes sense to me.